### PR TITLE
Update text padding for marquee and static media display

### PR DESCRIPTION
**Pull request by laiyichin**
#### WHAT
Update text padding for marquee and static media display

#### WHY
The padding was removed since the last wear compose update. This is to revert it back to how it was before pull/1487

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_default.png
Comment (luizgrp) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_default.png:1]: do you understand what was changed in this test? 

just wondering if this test was failing or it was just updated due to the image being generated in a different machine

File: media/ui/src/main/java/com/google/android/horologist/media/ui/components/animated/MarqueeTextMediaDisplay.kt [hunk 1]
```diff
@@ -69,7 +71,7 @@ public fun MarqueeTextMediaDisplay(
                 currentTitle ->
             MarqueeText(
                 text = currentTitle.orEmpty(),
-                modifier = Modifier.fillMaxWidth(0.7f),
+                modifier = Modifier.fillMaxWidth(0.7f).padding(top = 2.dp, bottom = .8.dp),
```
Comment (luizgrp) [inline media/ui/src/main/java/com/google/android/horologist/media/ui/components/animated/MarqueeTextMediaDisplay.kt:74]: Kdocs of [LoadingMediaDisplay](https://github.com/google/horologist/blob/772a08a052d6b666f96e6d3fdba46e20363a15b9/media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt#L40-L41) says that it matches the text of TextMediaDisplay and MarqueeTextMediaDisplay but the changes in this PR seem to be trying to make TextMediaDisplay and MarqueeTextMediaDisplay to match the loading shapes of LoadingMediaDisplay.

**Review by luizgrp — APPROVED** [review]

**Comment by yschimke** [comment]
@laiyichin Is this still planned?

**Comment by laiyichin** [comment]
> @laiyichin Is this still planned?

Ah yes, sorry! Let's get this one in first.

**Pull request merged**